testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -1230,6 +1230,35 @@ testModule('position', moduleConfig, () => {
 
         assert.strictEqual($content.position().top, event.target.position().top, 'overlay is positioned correctly');
     });
+
+    test('overlay content should have correct position when position.boundary changes to window', function(assert) {
+        const $overlay = $('<div>').appendTo('#qunit-fixture');
+        const $boundary = $('<div>').css({ width: 20, height: 30 }).appendTo('#qunit-fixture');
+        const widthContent = 25;
+        const heightContent = 35;
+
+        $overlay.dxOverlay({
+            visible: true,
+            width: widthContent,
+            height: heightContent,
+            animation: null,
+            position: {
+                collision: 'fit',
+                boundary: $boundary,
+            }
+        });
+
+        const overlay = $overlay.dxOverlay('instance');
+        const $overlayContent = overlay.$content();
+
+        assert.strictEqual($overlayContent.offset().top, $boundary.offset().top, 'top border of the content is correct');
+        assert.strictEqual($overlayContent.offset().left, $boundary.offset().left, 'left border of the content is correct');
+
+        overlay.option('position.boundary', window);
+
+        assert.roughEqual($overlayContent.offset().top, getHeight($(window)) / 2 - heightContent / 2, 0.51, 'content is in the center of the window vertically');
```

Comment:
let's calculate overlay content's middle coordinate, window's middle coordinate, save it to variables and assert equality of this variables. I hope it will make the assert more intuitive. Like:
```javascript
const contentOffset = $overlayContent.offset();
const contentCenterY = contentOffset.top + contentHeight / 2;
const windowCenterY = window.innerHeight / 2;
assert.roughtEqual(contentCenterY, windowCenterY, 'content is in the center of window vertically')
```

Reply:
Moved to variables